src/home/home.scss

```diff
@@ -323,18 +323,3 @@
     }
   }
 }
-
-@media screen and (max-width: 420px) {
```

Comment:
This should be there, no?  Was part of a previous PR?

Reply:
No, this part is from the previous PR. I'm not sure why it has been recommitted, but it should not be included in the code.

Reply:
Can you remove it then?